### PR TITLE
Fixed map colors

### PR DESCRIPTION
**Pull request by trainvoi**

### CODE REVIEW

File: common/country_definitions/rework.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ﻿NGF = { # Gray NGF
-	color = {105 105 105}
+	color = { 143  168  154 }
```
Comment (imperialist-x) [inline common/country_definitions/rework.txt:2]: Not needed, current color is already good

File: .metadata/metadata.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "id" : "",
+  "version" : "",
+  "supported_game_version" : "",
+  "short_description" : "",
+  "tags" : [],
+  "relationships" : [],
+  "game_custom_data" : {
+    "multiplayer_synchronized" : true
+  }
+}
```
Comment (imperialist-x) [inline .metadata/metadata.json:12]: what the hell is this file about?

File: common/country_definitions/rework.txt [hunk 3]
```diff
@@ -9,6 +9,17 @@
...
+	color = { 142  201  208 }
+	
+	country_type = recognized
+
+	tier = kingdom
+
+	cultures = { south_german}
+	capital = STATE_BAVARIA
+}
+
```
Comment (imperialist-x) [inline common/country_definitions/rework.txt:22]: This is cool

File: common/country_definitions/rework.txt [hunk 4]
```diff
@@ -30,7 +41,7 @@ TUR = {
 	capital = STATE_EASTERN_THRACE
 }
 JAB = {
-	color = { 120  150  162 }
+	color = { 255 204 45 }
```
Comment (imperialist-x) [inline common/country_definitions/rework.txt:44]: this looks bad

File: common/country_definitions/rework.txt [hunk 5]
```diff
@@ -40,7 +51,7 @@ JAB = {
 	capital = STATE_HAIL
 }
 PER = {
-	color = { 0 80 170 }
+	color = { 229  170  87 }
```
Comment (imperialist-x) [inline common/country_definitions/rework.txt:54]: color looks bad, current perisa color is cooler

File: common/country_definitions/rework.txt [hunk 6]
```diff
@@ -60,7 +71,7 @@ PRU = {
 	capital = STATE_BRANDENBURG
 }
 GER = {
-	color = { 23 43 59 } 
+	color = { 150 150 150 }
```
Comment (imperialist-x) [inline common/country_definitions/rework.txt:74]: not needed, current color is good

File: common/country_definitions/rework.txt [hunk 7]
```diff
@@ -81,7 +92,7 @@ GBR = {
 	capital = STATE_HOME_COUNTIES
 }
 WAL = {
-	color = { 204  126  0 }
+	color = { 161  126  128 }
```
Comment (imperialist-x) [inline common/country_definitions/rework.txt:95]: no just bad

File: common/country_definitions/rework.txt [hunk 8]
```diff
@@ -205,6 +216,18 @@ AUS = {
 	cultures = { south_german }
 	capital = STATE_AUSTRIA
 }
+
+KUK = {
+	color = { 124 40 30 }
```
Comment (imperialist-x) [inline common/country_definitions/rework.txt:221]: no, ottomans color is already red, this is conflicting

File: common/history/countries/chi - china.txt [hunk 9]
```diff
@@ -0,0 +1,47 @@
+﻿COUNTRIES = {
```
Comment (imperialist-x) [inline common/history/countries/chi - china.txt:1]: What did you change?

File: common/history/countries/dai - dai viet.txt [hunk 10]
```diff
@@ -0,0 +1,18 @@
+﻿COUNTRIES = {
```
Comment (imperialist-x) [inline common/history/countries/dai - dai viet.txt:1]: changes?

File: common/history/countries/jap - japan.txt [hunk 11]
```diff
@@ -1,6 +1,7 @@
 ﻿COUNTRIES = {
 	c:JAP = {
 		effect_starting_technology_tier_4_tech = yes
+		add_technology_researched = sericulture
```
Comment (imperialist-x) [inline common/history/countries/jap - japan.txt:4]: nice

File: common/history/countries/kor - korea.txt [hunk 12]
```diff
@@ -0,0 +1,19 @@
+﻿COUNTRIES = {
```
Comment (imperialist-x) [inline common/history/countries/kor - korea.txt:1]: Changes?

File: common/named_colors/redgb_country_color.txt [hunk 13]
```diff
@@ -1,3 +1,8 @@
 colors = {	
 	redgb = rgb { 0.686 0.059 0.059 } #175 15 15
+	feldgrau 	= 	rgb { 160 170 170 }
+	DDR_green	=	rgb { 150 70 70 }
```
Comment (imperialist-x) [inline common/named_colors/redgb_country_color.txt:4]: green? this is red, it also looks bad.. did you test it in-game?

File: common/named_colors/redgb_country_color.txt [hunk 14]
```diff
@@ -1,3 +1,8 @@
 colors = {	
 	redgb = rgb { 0.686 0.059 0.059 } #175 15 15
+	feldgrau 	= 	rgb { 160 170 170 }
```
Comment (imperialist-x) [inline common/named_colors/redgb_country_color.txt:3]: Nice

File: common/named_colors/redgb_country_color.txt [hunk 15]
```diff
@@ -1,3 +1,8 @@
...
+	DDR_green	=	rgb { 150 70 70 }
+	democratic_ger	=	rgb { 160 170 170 }
```
Comment (imperialist-x) [inline common/named_colors/redgb_country_color.txt:5]: good

File: common/named_colors/redgb_country_color.txt [hunk 16]
```diff
@@ -1,3 +1,8 @@
...
+	nazi_ger	=	rgb { 40 40 40 }
```
Comment (imperialist-x) [inline common/named_colors/redgb_country_color.txt:6]: good

File: common/named_colors/redgb_country_color.txt [hunk 17]
```diff
@@ -1,3 +1,8 @@
...
+	rust_brown	=	rgb { 124 40 30 }
```
Comment (imperialist-x) [inline common/named_colors/redgb_country_color.txt:7]: no

File: common/dynamic_country_map_colors/rework_map_color.txt [hunk 18]
```diff
@@ -21,4 +21,61 @@ imperial_japan = { #Red Imperial Japan
...
+	color = "nazi_ger"
+
+	possible = {
+		exists = c:GER
+		THIS = c:GER
+		coa_fascist_trigger = yes
+	}
+}
+
+austria_hungary = {
```
Comment (imperialist-x) [inline common/dynamic_country_map_colors/rework_map_color.txt:73]: no

**Review by imperialist-x — CHANGES_REQUESTED** [review]

File: .metadata/metadata.json (hunk 2 above)
Comment (trainvoi) [reply .metadata/metadata.json:12]: metadata so I could run the mod

File: common/country_definitions/rework.txt (hunk 1 above)
Comment (trainvoi) [reply common/country_definitions/rework.txt:2]: nah, vic2 gray NGF is better

File: common/dynamic_country_map_colors/rework_map_color.txt (hunk 18 above)
Comment (trainvoi) [reply common/dynamic_country_map_colors/rework_map_color.txt:73]: bruh, yellow AUS is cursed

File: common/country_definitions/rework.txt (hunk 4 above)
Comment (trainvoi) [reply common/country_definitions/rework.txt:44]: I want it to different from Ottoman Red

File: common/country_definitions/rework.txt (hunk 5 above)
Comment (trainvoi) [reply common/country_definitions/rework.txt:54]: disagree

**Pull request closed without merging**